setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup, find_packages
+setup(
+    name='substrait-consumer',
```

Comment:
I think this project has grown to a point where it's testing both producers and consumers.  maybe just substrait-tests? and change the folder to substrait_tests as well?  we can get feedback from others too.

Reply:
Yeah, probably we need to update the repo name too. Let’s decide a better name. +1

Reply:
cc @jacques-n @westonpace